### DISCUSSION TITLE
Help powering suits

### DISCUSSION THREAD

**Discussion by abduct3d** [lehjr/MachineMusePowersuits#91] [Q&A] [answered]
I am playing the 1.19.2 version, and I wanted to see if it was possible to charge my armor using IC2 Classic? I know in past versions of this mod you could charge using EU, but I can't get it to work on the current version. Is this a setting I have to enable? Turning on certain modules like Fortune on the power fist drains power wayy to fast.

**Comment by lehjr** [comment]
Currently there is no support for IC2 but I am in the process of looking into it, specifically trying to add it and test it before committing to it, possibly adding the ability to use their energy storage devices in the MPS armor and power fist. I'll know more in the next day or so.

**Comment by lehjr** [comment, marked as answer]
So turns out there isn't really any good way to add support back in without making it a hard dependency and unfortunately there is some intent behind that: https://github.com/TinyModularThings/IC2Classic/issues/856

**Reply by abduct3d** [reply]
That's unfortunate to hear, I appreciate your response and taking the time to look into it. Still one of my favorite mods,  I just gotta figure out a good way to power it :)
